src/plugins/blocknative.ts

```diff
@@ -1,32 +1,34 @@
-import Notify from 'bnc-notify';
+import BlocknativeSdk from 'bnc-sdk';
+import { InitializationOptions } from 'bnc-sdk/dist/types/src/interfaces';
 
-export const bnNotifySymbol = Symbol();
+export const bnSdkSymbol = Symbol();
 
-interface Options {
-  dappId: string;
-  networkId: number;
-  desktopPosition:
-    | 'bottomLeft'
-    | 'bottomRight'
-    | 'topLeft'
-    | 'topRight'
-    | undefined;
-}
-
-export const defaultOptions: Options = {
+export const defaultOptions: InitializationOptions = {
   dappId: process.env.VUE_APP_BLOCKNATIVE_DAPP_ID || '',
   networkId: Number(process.env.VUE_APP_NETWORK) || 1,
-  desktopPosition: 'bottomLeft'
+  onerror: error => {
+    console.log(`[Blocknative] encountered an error - ${error}`);
+  }
 };
 
 export default {
   install: app => {
-    const notifyInstance = Notify(defaultOptions);
+    const blocknative = new BlocknativeSdk(defaultOptions);
+
+    // filter out pending simulation events
+    blocknative
+      .configuration({
+        scope: 'global',
+        filters: [{ status: 'pending-simulation', _not: true }]
+      })
+      .catch(() => {
+        // swallow server timeout response error as we are not waiting on it
+      });
```

Comment:
This stuff is taken from `bnc-notify` seems like its needed to filter out simulation events.